romana_setup.md

```diff
@@ -0,0 +1,117 @@
+# The `romana-setup` tool
+
+The `romana-setup` tool is used to provision and install Romana using the selected installation environment, linux distribution and stack type.
+
+## Usage Summary
+
+```
+romana-setup: executes the ansible playbooks for installing romana
+Usage: romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] [action]
+       romana-setup [-n name] [-p platform] [-d distro] [-s stacktype] <action> [ansible-options]
+       romana-setup <h|--help>
+
+Stack Name:  user-defined stack name (default: $USER)
+Platforms:   aws (default), vagrant
+Distro:      ubuntu (default), centos
+Stack Types: devstack (default), kubernetes
+Actions:     install (default), uninstall
+```
+
+## Option Details
+
+### Name
+
+A name for the installation environment can be specified using the `-n` or `--name` option.
+The name must contain only ASCII letters and digits, and start with a letter.
+
+If not specified, the value of `$USER` is used.
+You should specify this option if you:
+- create multiple installs using `romana-setup`
+- do not have `USER` set to a valid value
+- prefer a different name for items created by `romana-setup`
+
+### Platform
+
+A platform can be specified using the `-p` or `--platform` option. `romana-setup` will use this to create hosts or virtual machines prior to installing the stack and Romana components.
```

Comment:
..., unless the 'static' option is chosen in which case no virtual machines will be created.